macaw-aarch32-symbolic/src/Data/Macaw/AArch32/Symbolic.hs

```diff
@@ -181,6 +198,71 @@ aarch32RegStructType :: CT.TypeRepr (MS.ArchRegStruct SA.AArch32)
 aarch32RegStructType =
   CT.StructRepr (MS.typeCtxToCrucible (FC.fmapFC MT.typeRepr aarch32RegAssignment))
 
+-- The following definitions for rN are tightly coupled to that of
+-- ArchRegContext for AArch32. Unit tests in the test suite ensure that they are
+-- consistent with regIndexMap (below).
+
+type GlobalsCtx = MS.CtxToCrucibleType (PC.MapCtx ToMacawTypeWrapper LAG.SimpleGlobalsCtx)
```

Comment:
Perhaps this definition should live near `RegContext` above?

Reply:
Why would that be? It's not exported, it's just used immediately below here.

Reply:
Ah, only in the sense that it's conceptually related to `RegContext` (given that `GlobalsCtx` is a specific subset of `RegContext`). That being said, I'm not too picky about this, so I'm fine with keeping it here as well.

Reply:
I added a comment explaining its relation to `RegContext`/`ArchRegStruct`.